### ISSUE TITLE
Fixed broken links reported from aggregate-report

### ISSUE THREAD

**Issue by sima-zhu** [Azure/azure-sdk-for-net#15503]
Remove locale from Security.md page. [Link](https://github.com/Azure/azure-sdk-for-net/blob/master/SECURITY.md)
  ```
  https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)
  https://www.microsoft.com/en-us/msrc/cvd
  https://www.microsoft.com/en-us/msrc/pgp-key-msrc
```

Broken pipeline.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=553433&view=logs&j=10327055-d4d6-50a6-a312-3d2cd4bd9cb9

**Comment by jsquire** [comment]
@sima-zhu:  Would you please be so kind as to provide additional context around which library this should be routed to?

**Comment by sima-zhu** [comment]
Updated the description.

**Comment by jsquire** [comment]
Fixed by #15527
